ui/packages/consul-ui/tests/acceptance/steps/dc/services/show/intentions/index-steps.js

```diff
@@ -0,0 +1,10 @@
+import steps from '../../../../steps';
+
+// step definitions that are shared between features should be moved to the
+// tests/acceptance/steps/steps.js file
+
+export default function(assert) {
+  return steps(assert).then('I should find a file', function() {
+    assert.ok(true, this.step);
+  });
+}
```

Comment:
Not sure who will be reviewing this, but if you aren't aware these files are unfortunately necessary, automatically generated, and never edited, but they need to exist.